src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -88,15 +88,19 @@
 _UPLOADED_YAML = "uploaded.yaml"
 _WS_MAX_BATCH_SIZE = 10000
 
-_AssemblyTuple = namedtuple(
-    "AssemblyTuple",
-    ["assembly_name", "host_assembly_dir", "container_internal_assembly_path"],
+# fields of _WSObjTuple
+_OBJ_NAME = "obj_name"
+_OBJ_DIR = "host_obj_dir"
+_CONTAINER_OBJ_PATH = "container_internal_obj_path"
```

Comment:
These are actually paths to files, not objects, but they get uploaded and linked to from the objects. Maybe s/obj/file/?

Reply:
> Maybe s/obj/file/?
I didn't get what you mean here.

Reply:
E.g. host_file_dir, container_internal_file_dir

Reply:
👍